adafruit_emc2101/__init__.py

```diff
@@ -295,7 +269,7 @@ def initialize(self):
     def check_status(self):
         """Read the status register and check for a fault indicated."""
         if self._status & emc2101_regs.STATUS_ALERT:
-            raise EMC2101LimitsException()
+            raise ValueError("Status alert")
```

Comment:
This is probably a `RuntimeError`, since `ValueError` doesn't make sense here based on the context